### PR TITLE
mainMenuFeature US21

### PR DESCRIPTION
**Pull request by huanhuiying**
This is the first version for the menu code where the task print and display the main menu and When a user enters an option 1, 2 or 0, the program will process the option accordingly is done

### CODE REVIEW

**Comment by huanhuiying** [comment]
Please review and take a look, so I can make changes for second draft

File: SimpCity.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    for i in range (len(main_menu_options)):
+        print("{}. {}".format(i+1,main_menu_options[i]))
+    print()
+
+def configure_menu(configure_menu_options):
+    for i in range (len(configure_menu_options)):
+       print("{}. {}".format(i+1,configure_menu_options[i]))
+    print()
+    
+print("Welcome, mayor of Simp City")
```
[the remark this answers is not in the document]
Comment (huanhuiying) [reply SimpCity.py:17]: ok noted, I will go and make the changes

[the remark this answers is not in the document]
Comment (huanhuiying) [reply SimpCity.py:17]: Changes have been made please review

[the remark this answers is not in the document]
Comment (amanda-chan) [reply SimpCity.py:17]: Looks good!

**Review by claris0911 — APPROVED** [review]
looks okay :)

**Review by amanda-chan — APPROVED** [review]
Function looks good!

**Review by meganpng — COMMENTED** [review]
Reviewed and looks good, but switch can be used to make it more efficient if you want

**Review by meganpng — APPROVED** [review]

**Comment by huanhuiying** [comment]
> Reviewed and looks good, but switch can be used to make it more efficient if you want

When I was writing the code, I thought of using the Switch function too, but when I went to checked, python does not have the Switch() function

**Comment by claris0911** [comment]
The menu also includes option 0 for exiting the game

**Comment by meganpng** [comment]
Please upload your test case file for future reference.

**Comment by huanhuiying** [comment]
[SimpCity US21 TDD.xlsx](https://github.com/huanhuiying/DevOps_Oct2021_Team-5/files/7687067/SimpCity.US21.TDD.xlsx)
Sorry, forgotten to upload it after the meeting, this is the TDD test case.

**Pull request merged**